modules/formulize/admin/save/relationship_settings_save.php

```diff
@@ -96,8 +96,8 @@ function addlink($form1_id, $form2_id, $relationship_id) {
 
     // write the link to the links table
     $writelink = "INSERT INTO " . $xoopsDB->prefix("formulize_framework_links") .
-        " (fl_frame_id, fl_form1_id, fl_form2_id, fl_key1, fl_key2, fl_relationship, fl_unified_display, fl_unified_delete, fl_common_value)".
```

Comment:
This needs reverting, we need the explicit value in the DB at all times, even though we're moving ahead with everything always being this way, and not worrying about the value, we want the data to be clean. (?) or have we/can we delete the field from the DB entirely and remove all references to it??